src/tools/r2rdump/UnwindInfo.cs

```diff
@@ -6,11 +6,47 @@
 
 namespace R2RDump
 {
+    public enum Registers
+    {
+        RAX = 0,
+        RCX = 1,
+        RDX = 2,
+        RBX = 3,
+        RSP = 4,
+        RBP = 5,
+        RSI = 6,
+        RDI = 7,
+        R8 = 8,
+        R9 = 9,
+        R10 = 10,
+        R11 = 11,
+        R12 = 12,
+        R13 = 13,
+        R14 = 14,
+        R15 = 15,
+    }
+
     struct UnwindCode
     {
+        public enum OpCodes
+        {
+            UWOP_PUSH_NONVOL = 0,
+            UWOP_ALLOC_LARGE,
+            UWOP_ALLOC_SMALL,
+            UWOP_SET_FPREG,
+            UWOP_SAVE_NONVOL,
+            UWOP_SAVE_NONVOL_FAR,
+            UWOP_EPILOG,
+            UWOP_SPARE_CODE,
+            UWOP_SAVE_XMM128,
+            UWOP_SAVE_XMM128_FAR,
+            UWOP_PUSH_MACHFRAME,
+            UWOP_SET_FPREG_LARGE,
+        }
+
         public byte CodeOffset { get; }
-        public byte UnwindOp { get; } //4 bits
-        public byte OpInfo { get; } //4 bits
+        public OpCodes UnwindOp { get; } //4 bits
+        public Registers OpInfo { get; } //4 bits
```

Comment:
The OpInfo is not always a register index. For example, for UWOP_ALLOC_LARGE, it indicates whether the frame offset is stored as 1 or two 16 bit values, for UWOP_SAVE_XMM128, it is an index of a xmm register, etc. 
So it should stay as a number and we can add OpCode specific conversion to string when you print it.
I can gather the per opcode details for you.

Reply:
Would info about the opcodes be in unwindamd64.cpp DumpUnwindInfo?

Reply:
@acmyu ah, yes, the printfs in the switch sounds like a good source for the info. The only thing it doesn't do is to report register names for the opcodes where the opinfo represents them. 
For UWOP_PUSH_NONVOL, UWOP_SAVE_NONVOL, UWOP_SAVE_NONVOL_FAR, the OpInfo is a general purpose register index (RAX, RBX, …). For UWOP_SAVE_XMM128 and UWOP_SAVE_XMM128_FAR, it is an index of the xmm register (0 for xmm0 etc).